pynucastro/templates/starkiller-cxx-microphysics/actual_rhs.H.template

```diff
@@ -67,32 +62,35 @@ void evaluate_rates(const burn_t& state, rate_eval_t& rate_eval) {
 
     Real rhoy = state.rho * state.y_e;
 
-    // initialize the screening factors to 1 -- this way if a rate is
-    // unscreened, when we multiply by the screening factor later, we
-    // don't zero the rate
-
-    for (int i = 1; i <= NumRates; ++i) {
-        rate_eval.unscreened_rates(i_scor, i) = 1.0_rt;
-        rate_eval.unscreened_rates(i_dscor_dt, i) = 0.0_rt;
-    }
-
     // Calculate Reaclib rates
 
     plasma_state_t pstate;
     fill_plasma_state(pstate, state.T, state.rho, Y);
 
     tf_t tfactors = evaluate_tfactors(state.T);
 
-    for (int i = 1; i <= NrateReaclib; ++i) {
-        Real rate;
-        Real drate_dt;
-        reaclib_evaluate(pstate, tfactors, i, rate, drate_dt);
-        rate_eval.unscreened_rates(i_rate, i) = rate;
-        rate_eval.unscreened_rates(i_drate_dt, i) = drate_dt;
+    if (jacobian == 1) {
+        for (int i = 1; i <= NrateReaclib; ++i) {
+            Real rate;
+            Real drate_dt;
+            reaclib_evaluate<1>(pstate, tfactors, i, rate, drate_dt);
```

Comment:
I recommend using constexpr ints here, rather than `0` and `1`, so we remember what they do in case there are future template parameters added.

Reply:
done